vllm/v1/engine/detokenizer.py

```diff
@@ -161,7 +161,7 @@ def __init__(self, tokenizer: PreTrainedTokenizerFast,
         prompt_suffix = request.prompt_token_ids
         prompt_len = len(prompt_suffix)
         if prompt_len > 4:
-            for i in range(4, max(prompt_len + 1, 32)):
+            for i in range(4, min(prompt_len + 1, 24)):
```

Comment:
Where does 24 come from? Can we use smaller numbers like 5?

Reply:
It's to try to find a small suffix to start from, or else it has to loop over the entire prompt below decoding the tokens one-by-one, which could be very long.

Reply:
So, does it mean we can't use smaller numbers?